.github/workflows/Unit-Test.yml

```diff
@@ -20,11 +20,19 @@ jobs:
       - name: List all changed package files
         uses: tj-actions/changed-files@v44
         if: steps.changed-ruby-files.outputs.any_changed == 'true'
+      - name: Unit Tests (x86_64)
         env:
           ALL_CHANGED_FILES: ${{ steps.changed-ruby-files.outputs.all_changed_files }}
-      - name: Unit Tests (x86_64)
         run: |
-            sudo docker run --rm -t satmandu/crewbuild:m90-x86_64 sudo -i -u chronos /bin/bash -c "
+            echo "ALL_CHANGED_FILES is ${ALL_CHANGED_FILES}" && \
+            export CHANGED_PACKAGES="$(echo ${ALL_CHANGED_FILES#packages/} | tr -d '.rb' | sort)" \
+            echo "CHANGED_PACKAGES is ${CHANGED_PACKAGES}" \
+            sudo docker run --rm \
+            -e ALL_CHANGED_FILES="${ALL_CHANGED_FILES}" \
+            -e CHANGED_PACKAGES="$(CHANGED_PACKAGES}" \
+            -t satmandu/crewbuild:m90-x86_64 \
```

Comment:
No need to run this unless changed package compatibility is all or x86_64?

Reply:
I think we still want to know that core functionality is not broken for arm...

Reply:
No need if the affected file changes are not including that architecture?

Reply:
Example:
There would be no need to run arm tests if the sbcl package was updated since it is only compatible with x86_64 and i686.